edge_computing/day_2_core_cnf_clusters/troubleshooting/telco-troubleshooting-mco.adoc

```diff
@@ -0,0 +1,20 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="telco-troubleshooting-mco"]
+= Machine Config Operator
+include::_attributes/common-attributes.adoc[]
+:context: telco-troubleshooting-mco
+
+toc::[]
+
+The Machine Config Operator provides useful information to cluster administrators and controls what is running directly on the bare-metal host.
+
+The Machine Config Operator differentiates between different groups of nodes in the cluster, allowing control plane nodes and worker nodes to run with different configurations.
+These groups of nodes run worker or application pods, which are called `MachineConfigPool` (`mcp`) groups.
+The same machine config is applied on all nodes or only on one MCP in the cluster.
+
+For more information about how and why to apply MCPs in a telco core cluster, see xref:../../../edge_computing/day_2_core_cnf_clusters/updating/telco-update-ocp-update-prep.adoc#telco-update-applying-mcp-labels-to-nodes-before-the-update_ocp-update-prep[Applying MachineConfigPool labels].
+
+For more information about the Machine Config Operator, see xref:../../../operators/operator-reference.adoc#machine-config-operator_cluster-operators-ref[Machine Config Operator].
+
+include::modules/telco-troubleshooting-mco-purpose.adoc[leveloffset=+1]
+include::modules/telco-troubleshooting-mco-apply-several-mcs.adoc[leveloffset=+2]
```

Comment:
Not sure why this is level 2 - shouldn't they be on the same level since applying machine config files isn't really a subset of the MCO purpose? (ignore if I am not understanding this correctly)

Reply:
It's a mistake, thanks for catching that :)